common/src/java/org/apache/hadoop/hive/ql/ErrorMsg.java

```diff
@@ -627,7 +627,9 @@
   REPL_INVALID_ARGUMENTS(40012, "Invalid arguments error : {0}.", true),
   REPL_INVALID_ALTER_TABLE(40013, "{0}Unable to alter table{1}", true),
   REPL_PERMISSION_DENIED(40014, "{0}org.apache.hadoop.security.AccessControlException{1}", true),
-  REPL_DISTCP_SNAPSHOT_EXCEPTION(40015, "SNAPSHOT_ERROR", true)
+  REPL_DISTCP_SNAPSHOT_EXCEPTION(40015, "SNAPSHOT_ERROR", true),
+  RANGER_AUTHORISATION_FAILED(40016, "Authorisation Failure while communicating to Ranger admin", true),
```

Comment:
error code is same. it should be different. Check if any of the existing errror codes that can be reused.